snapper.spec.in

```diff
@@ -100,8 +100,14 @@ This package contains snapper, a tool for filesystem snapshot management.
 %setup
 
 %build
+%if %{with coverage}
+# optimized code may confuse the coverage measurement, turn it off
+export CFLAGS="-g3"
+export CXXFLAGS="-g3"
```

Comment:
@lslezak There goes my "too much effort" argument :man_facepalming:

Reply:
... except it fails on Fedora, mysteriously. Multi platform builds are so much fun!!

Reply:
fixed!